Gemfile

```diff
@@ -28,4 +28,4 @@ gem "wdm", "~> 0.1.1", :platforms => [:mingw, :x64_mingw, :mswin]
 # do not have a Java counterpart.
 gem "http_parser.rb", "~> 0.6.0", :platforms => [:jruby]
 
-
```

Comment:
This was a missing dependency I ran into when building the site.

Reply:
nice catch.